addons/skin.estuary/xml/Home.xml

```diff
@@ -969,8 +907,12 @@
 					<width>480</width>
 					<height>110</height>
 					<top>100</top>
-					<onup>noop</onup>
-					<ondown>9000</ondown>
+					<onup>SetFocus(9000)</onup>
+					<onup>PageDown</onup>
+					<onup>PageDown</onup>
```

Comment:
oh youre right, in this direction SetFocus(x,xx) is fine. Will fix it for next sync.